ci/cloudbuild/builds/integration-daily.sh

```diff
@@ -23,7 +23,7 @@
 # `ci/cloudbuild/triggers` directory. "Manual" triggers exist only within the
 # GCB UI at the time of this writing. Users with the appropriate access can run
 # this build by hand with:
-#   `ci/cloudbuild/build.sh --distro fedora-36-bazel --build integration-daily --cloud cloud-cpp-testing-resources`
+#   `ci/cloudbuild/build.sh --distro fedora-37-bazel --build integration-daily --cloud cloud-cpp-testing-resources`
```

Comment:
I don't see the trigger change, but I presume you're doing that after this is in.

While you're there, it still has a "fedora-35" tag.

Reply:
Or nearly at the same time.  Fixed.

Reply:
It looks like the trigger went back to 35 instead of forward to 37.